### PR TITLE
Industry

### PR DESCRIPTION
**Pull request by hugovandepol**
v1 of industry track page

### CODE REVIEW

File: src/pages/industry.astro [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        <TimetableItem
+            time="9.30"
+            title="Registration"
+        />
+        <TimetableItem
+            time="10.00"
+            title="Opening"
+            speaker="Hugo van de Pol"
+        />
+        <TimetableItem
```
Comment (jdonszelmann) [inline src/pages/industry.astro]: inconsistent indentation?

**Comment by jdonszelmann** [comment]
looks good :)

File: src/pages/industry.astro (hunk 1 above)
Comment (erikjee) [reply src/pages/industry.astro]: Fixed by hugo in https://github.com/rustnl/rustnl2025/pull/25/commits/66e5504b2d77e92cb4147633dc033a431215036e

**Comment by erikjee** [comment]
@jdonszelmann I noticed yarn.lock is also included in this PR.  Is that okay? 
I just merged main, so you can press the button if everything looks good to you.

**Pull request merged**